### DISCUSSION TITLE
go.d plugin doesnt come installed by default on netdata container

### DISCUSSION THREAD

**Discussion by PedroVilla97** [netdata/netdata#14078] [Q&A]
Hello all. I have one netdata docker instance spin up (caprover one) and it only comes with the node and python plugins by default. 
Is there any way of installing the go.d plugin by hand?
Thank you all

**Comment by ilyam8** [comment]
Hi 👋 I think you need to contact CapRover support or raise an issue on GitHub.

My understanding is that this is their image: https://github.com/caprover/netdata-docker

> Based on https://github.com/titpetric/netdata with the exception that this is built as an multiarch image

It is based not on the [official image ](https://hub.docker.com/r/netdata/netdata), but on https://github.com/titpetric/netdata that is not maintained anymore (read as not the latest stable Netdata with who knows what changes)

> This repository has been archived by the owner before Nov 9, 2022. It is now read-only.

**Reply by PedroVilla97** [reply]
Thank you very much!
